### PR TITLE
 LookupArrayService task

### PR DESCRIPTION
**Pull request by VasiliZas**

### CODE REVIEW

File: homework/Vasili_Zasinets/src/test/java/com/tutrit/java/quickstart/homework/lookarrayservice/LookupArrayServiceTest.java [hunk 1]
```diff
@@ -0,0 +1,40 @@
+package com.tutrit.java.quickstart.homework.lookarrayservice;
+
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+
+public class LookupArrayServiceTest {
+    LookupArrayService lookupArrayService = new LookupArrayService();
```
Comment (max-java) [inline homework/Vasili_Zasinets/src/test/java/com/tutrit/java/quickstart/homework/lookarrayservice/LookupArrayServiceTest.java:8]: try test with int[] array = {};

File: homework/Vasili_Zasinets/src/main/java/com/tutrit/java/quickstart/homework/lookarrayservice/LookupArrayService.java [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+            }
+        }
+        log.info("arrayMin = {}", arrayMin);
+        return arrayMin;
+
+    }
+
+    public int indexOfMax(int[] array) {
+        int indexMax = -1;
+        if (array == null) return -1;
```
Comment (max-java) [inline homework/Vasili_Zasinets/src/main/java/com/tutrit/java/quickstart/homework/lookarrayservice/LookupArrayService.java:36]: That's ok, but as you've knew exceptions already I suggest that you take a look at lamaconda's enterprise approach for this check.

**Review by max-java — CHANGES_REQUESTED** [review]
please, use alt + shift + ctrl + L to align code, and run sonar verification before merge. 
also, minor comments in tests that requires code changes.

File: homework/Vasili_Zasinets/src/test/java/com/tutrit/java/quickstart/homework/lookarrayservice/LookupArrayServiceTest.java (hunk 1 above)
Comment (VasiliZas) [reply homework/Vasili_Zasinets/src/test/java/com/tutrit/java/quickstart/homework/lookarrayservice/LookupArrayServiceTest.java:8]: fixed

File: homework/Vasili_Zasinets/src/main/java/com/tutrit/java/quickstart/homework/lookarrayservice/LookupArrayService.java (hunk 2 above)
Comment (VasiliZas) [reply homework/Vasili_Zasinets/src/main/java/com/tutrit/java/quickstart/homework/lookarrayservice/LookupArrayService.java:36]: fixed.

**Review by KaydunovDenis — APPROVED** [review]
No description provided.

**Review by PavelBoPavel — APPROVED** [review]

**Review by max-java — APPROVED** [review]
I suggest adding tests on exception cases, otherwise lgfm

**Pull request merged**